lib/collections.js

```diff
@@ -41,7 +55,9 @@ var uniqueWords = [];
 
 // The longest and shortest word in the paragraph
 var longAndShort = {};
-
+for (var i = 0, i < words.length; ++1) {
```

Comment:
This is not what the question was asking. There was no reason to write anything here.